cmd/keytransparency-client/cmd/hammer.go

```diff
@@ -41,9 +44,11 @@ func init() {
 
 	RootCmd.AddCommand(hammerCmd)
 
-	hammerCmd.Flags().IntVar(&maxWorkers, "workers", 1, "Number of parallel workers")
+	hammerCmd.Flags().StringVar(&testTypes, "types", "batch,write,read,audit", "Types of stress tests to run")
+	hammerCmd.Flags().IntVar(&qps, "qps", 100, "Numer of requests a second")
+	hammerCmd.Flags().IntVar(&pageSize, "batch", 10, "Number of things to do at once")
+	hammerCmd.Flags().IntVar(&maxWorkers, "workers", 1000, "Number of parallel workers")
```

Comment:
Could you be a bit more specific on what the work of the workers is? E.g., RPC-sending workers.